1.10/Dockerfile.rhel7

```diff
@@ -32,7 +32,10 @@ LABEL summary="$SUMMARY" \
 ENV NGINX_CONFIGURATION_PATH=/opt/app-root/etc/nginx.d
 ENV NGINX_DEFAULT_CONF_PATH=/opt/app-root/etc/nginx.default.d
 
-RUN yum install -y yum-utils && \
+# To use subscription inside container yum command has to be run first (before yum-config-manager)
+# https://access.redhat.com/solutions/1443553
+RUN yum repolist > /dev/null && \
```

Comment:
I get same output on machine with subscription.
But I seems that even that the `yum install yum-utils` doesn't create anything, it updates repositories. So I don't know why it is not working.

Older `s2i-base` (without yum-utils) doesn't need `yum clean all` and first `yum search` finds rh-nginx18.